### ISSUE TITLE
Keep going | We have more stars than rms-open-letter

### ISSUE THREAD

**Issue by imachug** [rms-support-letter/rms-support-letter.github.io#1031]
There may be fewer people on our signature list, but we have more stars--this means even though many people are afraid of talking about their support for Stallman in public, they still support us via GitHub repository stars. Good job everyone, we'll win in the end!

**Comment by shenlebantongying** [comment]
Actually there are probably more than 1k now, this repo didn't switch to  `index.html` yet, so currently it shows the number from several hours ago.

**Comment by TimurC1** [comment]
We will fight for truth!

**Comment by a1batross** [comment]
The point is not the stars, but signatures.

Forget about stars.

**Comment by TimurC1** [comment]
> The point is not the stars, but signatures.
> 
> Forget about stars.

Most of people supporting couldn't pull request, check pull requests

**Comment by imachug** [comment]
> The point is not the stars, but signatures.
> 
> Forget about stars.

No, the point is to show our opinion. People are afraid of losing their job, especially during the pandemic, and we can't really expect them all to make PRs which may attract attention.

**Comment by a1batross** [comment]
@imachug I don't know how people can "support" someone but being afraid at same time. I know the reason but once in a life, it's possible to do this publicly. Same on people, who prefer signing with nicknames.
Eh... maybe I'm too strict, sorry %)

@TimurC1 that's why there is another issue: allow sending signatures through mail. Everyone have mail.

**Comment by ExceptionallyHandsome** [comment]
> People are afraid of losing their job, especially during the pandemic, and we can't really expect them all to make PRs which may attract attention.

So may stars. Starrers' names are recorded as well, and it's no trouble for a sinister employer to check them.

**Comment by imachug** [comment]
@ExceptionallyHandsome You can remove a star when you find out someone is searching for a reason to fire you. You can't unmerge a pull request.

**Comment by d-sfounis** [comment]
Russia stronk

**Comment by podvornyakva** [comment]
> @TimurC1 that's why there is another issue: allow sending signatures through mail. Everyone have mail.

By my oppinion this kind of signing demonstrate that, there is alot peope able, not only figurout with github, and how it works, but also create those own staf as a piece of big project. Not just "click like button" or "fingerprint iphone" or send word "yes" by email. They are can, and they are DO more. Through this they are CAN, and should directly participate in FS community decisions.

**Comment by cristeigabriel** [comment]
> > The point is not the stars, but signatures.
> > Forget about stars.
> 
> Most of people supporting couldn't pull request, check pull requests

Perhaps this should be more the reason to pull for linking #837 in the README(s).

**Comment by avoidr** [comment]
I'm still on the fence whether to sign or not. Of course I support it, but it may mean the end of your career before it even started. On the other hand, it might work as a filter; you'll get hired either by apolitical employers who get shit done, or you get hired by employers with maturity and reason, avoiding crazy fanatics. Surely that might actually save you some trouble.

We've also had more signatures than the open letter for some hours already!

**Comment by imachug** [comment]
@avoldr Really? Our site lists 1434, theirs lists 2018 signatures.

**Comment by avoidr** [comment]
Huh, I was counting contributors… I guess I don't know what a contributor is, then.

**Comment by podvornyakva** [comment]
In fact there is only 1840 commits. Just a half hour ago it was spare. Карусели.

**Comment by avoidr** [comment]
How do they get over 2000 signatures in less than 2000 commits and much less than 2000 contributors?

**Comment by imachug** [comment]
They say some people contributed via e-mail and direct messages.

**Comment by nukeop** [comment]
>  Of course I support it, but it may mean the end of your career before it even started.

You really think anyone is going to dig through the internet to see if you've signed a random letter 4 years ago?

**Comment by avoidr** [comment]
>You really think anyone is going to dig through the internet to see if you've signed a random letter 4 years ago?

Well, I don't know, man. Would you have thought there would be people gang-beating an old man on the ground for being ugly?

**Comment by podvornyakva** [comment]
> contributed via e-mail

To where, i should ask? Around github repository? Those are not CONTRIBUTORS. Those not iven a hub users. Then what for they are use github? Change.org is much more easy way to collect signs. Okey. Let's go further then. We shold start public vote on most poular social platforms.

**Comment by a1batross** [comment]
Friendly reminder: cancel culture works because you're afraid of it.

**Comment by avoidr** [comment]
Signed.

Here is a related anecdote:
"If I pretend not to care that they bully me, maybe they will get bored of bullying me". They never get bored. It makes them feel powerful, but level up and they lose their sense of power.

Thank you.

**Comment by SpheeresX** [comment]
As of `March 28 2021, 11:14 PM CEST` this letter actually has 746 more signatures than rms-open-letter!

**Comment by guoxiangyang** [comment]
2021-03-30 15:00:00  :   4218(support) vs 2855(remove)
keep going!!!
